jina/serve/runtimes/gateway/http/app.py

```diff
@@ -107,6 +108,19 @@ async def _shutdown():
             }
         )
 
+        @app.get(
+            path='/',
+            summary='Get the health of Jina service',
+            response_model=JinaHealthModel,
+        )
+        async def _status():
```

Comment:
Please change the name of this `method` since there is one already existing. or change the one of the one already existing

Reply:
yes my bad